start_benchmark.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+PHP_COMMAND="php -n ${PHP_OPTIONS}"
+NODEJS_COMMAND='node'
+
+AERYS_COMMAND="./aerys/vendor/bin/aerys -w 1 -c ./aerys/server.php --worker-args=\"-n\""
```

Comment:
@kelunik not sure that I understand what you mean. May following example will put more light about `-n` option:
```
➜  async-http git:(tarvis-ci-integration) ✗ php -n -dextension_dir=/Users/torinaki/www/htdocs/async-http/php-ext -v 
PHP 7.0.18 (cli) (built: Apr 25 2017 02:53:38) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
➜  async-http git:(tarvis-ci-integration) ✗ php -n -dextension_dir=/Users/torinaki/www/htdocs/async-http/php-ext -c php/default-opcache.ini -v
PHP 7.0.18 (cli) (built: Apr 25 2017 02:53:38) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
    with Zend OPcache v7.0.13, Copyright (c) 1999-2016, by Zend Technologies
```
`-n` - removes all configurations
`-c` - specify my own configuration
`-dextension_dir` - need this to specify correct path of all extensions otherwise PHP can't find `opcache.so` library